src/main/java/org/ibp/api/rest/labelprinting/ObservationDatasetLabelPrinting.java

```diff
@@ -240,7 +240,9 @@ public List<LabelType> getAvailableLabelTypes(final LabelsInfoInput labelsInfoIn
 
 		final List<MeasurementVariable> plotVariables = this.middlewareDatasetService.getObservationSetVariables(
 			plotDatasetId,
-			Arrays.asList(VariableType.EXPERIMENTAL_DESIGN.getId(), VariableType.GERMPLASM_DESCRIPTOR.getId()));
+			Arrays.asList(VariableType.EXPERIMENTAL_DESIGN.getId(), VariableType.GERMPLASM_DESCRIPTOR.getId(),
```

Comment:
@cuenyad anything we have to do with this FIXME now? 
If not, please improve the comment and add //FIXME